lldb/test/API/lang/swift/clangimporter/objc_baseclass/TestSwiftObjCBaseClassMemberLookup.py

```diff
@@ -0,0 +1,18 @@
+import lldb
+from lldbsuite.test.lldbtest import *
+from lldbsuite.test.decorators import *
+import lldbsuite.test.lldbutil as lldbutil
+
+class TestSwiftObjCBaseClassMemberLookup(TestBase):
+    NO_DEBUG_INFO_TESTCASE = True
+    @skipUnlessDarwin
+    @swiftTest
+    def test(self):
+        """Test accessing a static member from a member function"""
+        self.build()
+        lldbutil.run_to_source_breakpoint(
+            self, 'break here', lldb.SBFileSpec('main.swift')
+        )
+
+        # dwim-print start by checking whether a member 'A' exists in 'self'.
```

Comment:
In this dwim-print will fallback to expression evaluation, because `A` isn't a frame variable.

Reply:
so this is the same as `expression A.shared`.

Reply:
As far as the result is concerned, yes. But the code path I'm specifically checking here is the failing check for a member 'A' in `self`, which would have triggered a fallback, which will assert in future versions of the test suite.